SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -335,4 +337,11 @@ var splunkRumInitializeCalledTime = Date()
         integrateWebViewWithBrowserRum(view: view)
     }
 
+    /**
+       This check is to determine whether the splunkrum library has been initialized
+     */
+    public class func isInitialized() -> Bool? {
```

Comment:
Please adjust the unit tests so that before the call to `initialize`, `isInitialized` returns false, and then that the call to `initialize` returns true, and that then `isInitialized` also returns true.

Reply:
I will add unit tests.